xkeysnail/__init__.py

```diff
@@ -113,7 +114,7 @@ def cli_main():
         sys.exit(1)
 
     # Make sure that user have root privilege
-    if not has_access_to_uinput():
+    if not has_access_to_uinput() or os.getuid() != 0:
```

Comment:
I don't follow this... surely access to uinput is ALL that matters here, what is the extra check doing?  We might have access without being root - it's east to configure that.

Reply:
At the time, the goal was probably to make it more verbose and clear why it couldn't be executed, it's been that way for some time.

As it doesn't affect the remap loop, I don't think it's critical in terms of performance.

```    @staticmethod
    def check_if_access_to_uinput():
        """Staticmethod to check if _uinput is accessible."""
        if _UINPUT:
            return True
        else:
            log_msg("Failed to open `uinput` in write mode.")
            if os.getuid() != 0:
                log_msg("Make sure that you have executed keyscrew with root privileges.")
                log_msg("Such as: sudo keyscrew -c config.py")
            sys.exit(1)
```

Reply:
Much of my code review (and hence the questions) is about clarity, not just performance. :)  I want to 100% understand every line of code.